src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -772,20 +769,6 @@ public static void onCommandRegister(CommandDispatcher<CommandSourceStack> dispa
         MinecraftForge.EVENT_BUS.post(event);
     }
 
-    public static net.minecraftforge.event.entity.EntityEvent.Size getEntitySizeForge(Entity entity, Pose pose, EntityDimensions size, float eyeHeight)
-    {
-        EntityEvent.Size evt = new EntityEvent.Size(entity, pose, size, eyeHeight);
-        MinecraftForge.EVENT_BUS.post(evt);
-        return evt;
-    }
-
-    public static net.minecraftforge.event.entity.EntityEvent.Size getEntitySizeForge(Entity entity, Pose pose, EntityDimensions oldSize, EntityDimensions newSize, float newEyeHeight)
-    {
-        EntityEvent.Size evt = new EntityEvent.Size(entity, pose, oldSize, newSize, entity.getEyeHeight(), newEyeHeight);
-        MinecraftForge.EVENT_BUS.post(evt);
-        return evt;
-    }
```

Comment:
These two methods should be reinstated and deprecated for removal as removing them is a breaking change (and also definitely [affects](https://gist.github.com/Pewmaymen/2d9ae4bacb9a128aae8f914f8008b010) mods)